Classes/Swisscom/Neos/CaseInsensitiveUrls/Aspect/LowercaseRoutePartAspect.php

```diff
@@ -27,8 +27,14 @@ public function initializeObject() {
 	 * @return void
 	 */
 	public function convertRoutePartToLowerCase(\TYPO3\Flow\AOP\JoinPointInterface $joinPoint) {
-		$lowerCaseRequestPath = strtolower($joinPoint->getMethodArgument('requestPath'));
+		$requestPath = $joinPoint->getMethodArgument('requestPath');
 
+		if (strpos($requestPath, 'user')) {
```

Comment:
When the NodeTree is built the API calls do not contain the at sign, thus not fixing the actual issue. 

Actual URLs with 'user' in them will be lowercase anyway after the migration's been run. The validation won't allow users to create new URLs containing uppercase letters.